app/src/main/java/com/rud/movieapp/di/RepositoryModule.kt

```diff
@@ -0,0 +1,16 @@
+package com.rud.movieapp.di
+
+
+import com.rud.data.MovieWebService
+import com.rud.data.repository.MovieRepositoryImpl
+import com.rud.domain.MovieRepository
+import dagger.Module
+import dagger.Provides
+
+@Module
+class RepositoryModule {
+    @Provides
+    fun provideMovieRepo(movieServices: MovieWebService): MovieRepository =
```

Comment:
singleton. As it defined on the main component level.

Reply:
Fixed.